guess-the-number/guess_the_number/guess.py

```diff
@@ -137,10 +144,39 @@ async def start_new_game(q: Q):
     await q.page.save()
 
 
+async def toggle_theme(q):
+
+    if not q.client.initialized:
```

Comment:
This seems to be odd `toggle_theme` should not also initialize the app itself, it should only switch theme. It would be also good to add the theme switch handler among other handlers - into `run_app` function.